### PR TITLE
[IF super res] correctly normalize PIL input

### PR DESCRIPTION
**Pull request by williamberman**
re: https://github.com/huggingface/diffusers/issues/3391

### CODE REVIEW

File: src/diffusers/pipelines/deepfloyd_if/pipeline_if_img2img_superresolution.py [hunk 1]
```diff
@@ -759,7 +759,7 @@ def preprocess_image(self, image: PIL.Image.Image, num_images_per_prompt, device
             image = [image]
 
         if isinstance(image[0], PIL.Image.Image):
-            image = [np.array(i).astype(np.float32) / 255.0 for i in image]
+            image = [np.array(i).astype(np.float32) / 175.0 - 1.0 for i in image]
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/deepfloyd_if/pipeline_if_img2img_superresolution.py:762]: Where does the 175 come from? Bit of a weird number as 255 / 2 is more like ~128

Comment (williamberman) [reply src/diffusers/pipelines/deepfloyd_if/pipeline_if_img2img_superresolution.py:762]: I shouldn't do math in my head when sleep deprived

Comment (williamberman) [reply src/diffusers/pipelines/deepfloyd_if/pipeline_if_img2img_superresolution.py]: fixed!

**Review by patrickvonplaten — APPROVED** [review]
Thanks for correcting!

File: src/diffusers/pipelines/deepfloyd_if/pipeline_if_img2img_superresolution.py (hunk 1 above)
Comment (patrickvonplaten) [reply src/diffusers/pipelines/deepfloyd_if/pipeline_if_img2img_superresolution.py]: All good, that's why we have the review system ;-)

**Pull request merged**